go.mod

```diff
@@ -173,9 +173,8 @@ require (
 	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/pyroscope-io/dotnetdiag v1.2.1 // indirect
 	github.com/quic-go/qpack v0.4.0 // indirect
-	github.com/quic-go/qtls-go1-19 v0.3.3 // indirect
-	github.com/quic-go/qtls-go1-20 v0.2.3 // indirect
-	github.com/quic-go/quic-go v0.36.4 // indirect
+	github.com/quic-go/qtls-go1-20 v0.3.2 // indirect
```

Comment:
The downgrade here makes me suspicious, but `github.com/quic-go` doesn't need `github.com/quic-go/qtls-go1-20` when compiled for 1.21, so it might not matter...

Reply:
IDK, it happened when I updated to go-libp2p v0.30.0 🤷